catalog/admin/admin-poller-service/src/main/java/org/codice/ddf/catalog/admin/poller/AdminPollerServiceBean.java

```diff
@@ -109,7 +107,6 @@ public AdminPollerServiceBean(ConfigurationAdmin configurationAdmin, org.osgi.se
 
     public void init() {
         serviceFactoryFilter = getServiceFactoryFilterProperties();
-        serviceFilter = getServiceFilterProperties();
```

Comment:
✏️ Is this `getServiceFilterProperties()` method used anywhere? Can it be removed?